lib/ContractInteract/BrandedToken.js

```diff
@@ -0,0 +1,393 @@
+      );
+      return Promise.reject(err);
+    }
+
+    const tx = await this.acceptStakeRequestRawTx(stakeRequestHash, r, s, v);
+    return Utils.sendTransaction(tx, txOptions);
+  }
+
+  /**
+   * Raw transaction for accept stake request.
+   *
+   * @param stakeRequestHash Stake request hash.
+   * @param r R of signature received from worker.
+   * @param s s of signature received from worker.
+   * @param v v of signature received from worker.
+   *
+   * @return Promise<Object> Raw transaction object.
+   */
+  acceptStakeRequestRawTx(stakeRequestHash, r, s, v) {
+    if (!stakeRequestHash) {
+      const err = new TypeError(`Invalid stakeRequestHash: ${stakeRequestHash}.`);
+      return Promise.reject(err);
+    }
+    if (!r) {
+      const err = new TypeError(`Invalid r of signature: ${r}.`);
+      return Promise.reject(err);
+    }
+    if (!s) {
+      const err = new TypeError(`Invalid s of signature: ${s}.`);
+      return Promise.reject(err);
+    }
+    if (!v) {
+      const err = new TypeError(`Invalid v of signature: ${v}.`);
+      return Promise.reject(err);
+    }
+
+    return Promise.resolve(
+      this.contract.methods.acceptStakeRequest(
+        stakeRequestHash,
+        r,
+        s,
+        v,
+      ),
+    );
+  }
+
+  /**
+   * Lift restriction for given list of addresses.
+   *
+   * @param addresses Addresses for which to lift restrictions.
+   * @param txOptions Transaction options.
+   *
+   * @return {Promise<Object>} Promise that resolves to transaction receipt.
+   */
+  async liftRestriction(addresses, txOptions) {
+    if (!txOptions) {
+      const err = new TypeError(`Invalid transaction options: ${txOptions}.`);
+      return Promise.reject(err);
+    }
+    if (!Web3.utils.isAddress(txOptions.from)) {
+      const err = new TypeError(
+        `Invalid from address ${txOptions.from} in transaction options.`,
+      );
+      return Promise.reject(err);
+    }
+
+    const tx = await this.liftRestrictionRawTx(addresses);
+    return Utils.sendTransaction(tx, txOptions);
+  }
+
+  /**
+   * Raw tx for lift restriction.
+   *
+   * @param addresses Addresses for which to lift restrictions.
+   *
+   * @return Promise<Object> Raw transaction object.
+   */
+  liftRestrictionRawTx(addresses) {
+    if (!addresses || addresses.length === 0) {
+      const err = new TypeError(
+        `At least one addresses must be defined : ${addresses}`,
+      );
+      return Promise.reject(err);
+    }
+    return Promise.resolve(this.contract.methods.liftRestriction(addresses));
+  }
+
+  /**
+   * Checks if given address is unrestricted.
+   *
+   * @param address Address needs to be checked.
+   *
+   * @returns {Promise<boolean>} Promise that resolves to `true` if
+   * unrestricted..
+   */
+  isUnrestricted(address) {
+    return this.contract.methods
+      .isUnrestricted(address)
+      .call();
+  }
+
+  /**
+   *  This rejects stake request, should be called by organization worker.
```

Comment:
```suggestion
   * This rejects a stake request, must be called by organization worker.
```
**must** be called by worker or am I wrong? 🤔